app/models/link_report.rb

```diff
@@ -17,7 +17,7 @@ def create_batch
 private
   
   def list_contents(list_contents)
-    list_contents.map { |content_item| content_item.fetch(:href) }
+    list_contents.map { |content_item| content_item.fetch(:href) } unless list_contents.nil?
```

Comment:
Is this change related to adding `request_broken_links`? If not, can you squash it up to the previous commit?